build.gradle

```diff
@@ -111,6 +111,8 @@ dependencies {
     compileOnly group: 'org.projectlombok', name: 'lombok', version: lombokVersion // not needed at runtime
     compileOnly group: 'org.apache.flink', name: 'flink-streaming-scala_2.11', version: flinkVersion // provided by application
     compileOnly group: 'org.apache.flink', name: 'flink-table_2.11', version: flinkVersion // provided by application
+    compileOnly group: 'org.apache.flink', name: 'flink-json', version: flinkVersion
```

Comment:
So flick version is consistent across all components?

Reply:
yes